resources/views/secretariat/document/list.blade.php

```diff
@@ -0,0 +1,51 @@
+@extends('layouts.app')
+
+@section('title')
+<a href="#!" class="breadcrumb">@lang('admin.admin')</a>
+<a href="#!" class="breadcrumb">@lang('admin.document_requests')</a>
+@endsection
+@section('secretariat_module') active @endsection
+
+@section('content')
+<div class="row">
+    <div class="col s12">
+        <div class="card">
+            <div class="card-content">
+                <span class="card-title">@lang('admin.document_requests')</span>
+                <table>
+                    <thead>
+                        <tr>
+                            <th>@lang('user.name')</th>
+                            <th>@lang('user.neptun')</th>
+                            <th>@lang('document.request')</th>
+                            <th>@lang('checkout.date')</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        @foreach ($requests as $request)
+                        <tr>
+                            <td>
+                                {{ $request->user->name ?? 'Name not found' }}
+                            </td>
+                            <td>
+                                {{ $request->user->educationalInformation->neptun ?? 'Neptun code not found'}}
+                            </td>
+                            <td>
+                                @if ($request->document_type === 'StatusCertificate')    
+                                    @lang('document.status-cert')
+                                @else
+                                    {{ $request->document_type }}
+                                @endif
```

Comment:
You should use @lang('document.' . $request->document_type) and add the document types to the langugas files as keys.